indices/get_high_overlap_fields.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+import glob
+import os
+import pickle as pkl
+import shutil
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('in_dir', help='The pagerank results to be filtered')
+    parser.add_argument('out_dir', help='The location to store results with high overlap')
+    parser.add_argument('overlap_threshold',
+                        help='The minimum number of shared papers to include a field pair',
+                        default=1000,
+                        type=int)
+    args = parser.parse_args()
+
+    i = 0
+    for path in glob.glob(os.path.join(args.in_dir, '*.pkl')):
+        filename = os.path.basename(path)
+        try:
+            with open(path, 'rb') as in_file:
+                df = pkl.load(in_file)
+        except FileNotFoundError:
+            continue
+
+        if len(df) > OVERLAP_THRESHOLD:
```

Comment:
Does this work? I don't see `OVERLAP_THRESHOLD` defined anywhere - I would have thought you have to use `args.overlap_threshold` here instead

Reply:
Good catch, I swore I replaced all the instances but I was wrong